network/import-nodes.go

```diff
@@ -271,8 +274,11 @@ func buildNetworkNodesMap(dbNodes []models.NetworkNode) map[string]nodeInfo {
 			RuiSection:     ruiSection,
 		}
 
+		if !lib.SliceContains(emailsList, nn.Mail) {
+			emailsList = append(emailsList, nn.Mail)
+		}
```

Comment:
Technically this control should not be necessary: if we are adding a node to the nodesMap, the email should be already unique, and later when we validate the rows we check if the email is present as to not add duplication. The 'problem' here is that we could potentially have a nodesMap with a number of keys different from the length of the emailsList.

And for the sake of duplication control when validating rows, it doesn't matter if it matches once or more times, if there is a match it errors.